### PR TITLE
Issue 869

### PR DESCRIPTION
**Pull request by sowmiyasubramaniyan**
Issue fixed:#869
         This PR is about In footer page  tweets and the MailChimp box comes only when the APIs are enabled

### CODE REVIEW

File: resources/views/themes/default1/front/widgets/create.blade.php [hunk 1]
```diff
@@ -46,26 +46,48 @@
...
                     </div>
+                     @if($twitterStatus == 1)
 
                     <div class="col-md-4 form-group {{ $errors->has('allow_tweets') ? 'has-error' : '' }}">
                         <!-- last name -->
                         {!! Form::label('allow_tweets',Lang::get('message.allow_tweets'),['class'=>'required']) !!}
                         {!! Form::select('allow_tweets',[1=>'Yes',0=>'No'],null,['class' => 'form-control']) !!}
 
                     </div>
+                    @endif
```
Comment (RafficMohammed) [inline resources/views/themes/default1/front/widgets/create.blade.php:57]: Use else if and just mention $twitterStatus. By default $twitterStatus is 1.

File: resources/views/themes/default1/front/widgets/create.blade.php [hunk 2]
```diff
@@ -46,26 +46,48 @@
...
+                    @if( $mailchimpStatus == 1)
                     <div class="col-md-4 form-group {{ $errors->has('allow_mailchimp') ? 'has-error' : '' }}">
                         <!-- last name -->
                         {!! Form::label('allow_mailchimp',Lang::get('message.allow_mailchimp'),['class'=>'required']) !!}
                         {!! Form::select('allow_mailchimp',[1=>'Yes',0=>'No'],null,['class' => 'form-control']) !!}
 
                     </div>
+                    @endif
+                    @if( $mailchimpStatus == 0)
+                   <div class="col-md-4 form-group {{ $errors->has('allow_mailchimp') ? 'has-error' : '' }}">
```
Comment (RafficMohammed) [inline resources/views/themes/default1/front/widgets/create.blade.php:75]: Same applicable here

File: resources/views/themes/default1/front/widgets/edit.blade.php [hunk 3]
```diff
@@ -46,27 +46,48 @@
...
+                    @if($twitterStatus == 1)
 
                     <div class="col-md-4 form-group {{ $errors->has('allow_tweets') ? 'has-error' : '' }}">
                         <!-- last name -->
                         {!! Form::label('allow_tweets',Lang::get('message.allow_tweets'),['class'=>'required']) !!}
                         {!! Form::select('allow_tweets',[1=>'Yes',0=>'No'],null,['class' => 'form-control']) !!}
 
                     </div>
+                    @endif
+                      @if($twitterStatus == 0)
```
Comment (RafficMohammed) [inline resources/views/themes/default1/front/widgets/edit.blade.php:59]: Follow the same steps what I mentioned earlier

**Review by priyadarshini2101 — APPROVED** [review]
Tested the pr with existing db on server.

Before fix
In API Keys module, If Either Mailchimp and Twitter toggle button enabled or disabled, Still in the Footer page the options are available for the Mailchimp and twitter.

After fix

Mailchimp  and twitter are disabled.

jul14mailchimp1

In Widget module of footer page options for mailchimp and twitter is hiding.

jul14widget1

Mailchimp and twitter are enabled.

jul14mailchimp

In widget module of footer page. the options for mailchimp and twitter are showing.

jul14widget

Note: Checked only for the pointed one....

**Review by sandesh556 — APPROVED** [review]

**Review by RafficMohammed — APPROVED** [review]

**Review by Sheetal34 — APPROVED** [review]
**Tested pr with existing db locally:**
 
Before fix:
In API Keys module, if Mailchimp and Twitter toggle button disabled, still in the Footer page,  the options are available for the Mailchimp and twitter.
image
image

After fix:
 If Mailchimp and Twitter toggle button disabled, in the Footer page,  the options are disabled  for the Mailchimp and twitter.
image
image

**Pull request merged**